components/builder-api/src/server/resources/origins.rs

```diff
@@ -1529,7 +1529,8 @@ fn download_content_as_file(content: &[u8], filename: String) -> HttpResponse {
             filename,
         )
         .header(http::header::CACHE_CONTROL, headers::NO_CACHE)
-        .body(Bytes::from(content))
+        .body(Bytes::from(content.to_vec())) // TODO: 0.5.4 no longer implements From<&'a [u8], only
+                                             // 'static
```

Comment:
It looks like this [issue](https://github.com/tokio-rs/bytes/issues/139) discusses the removal. I think the thing to use is [`copy_from_slice`](https://docs.rs/bytes/0.5.4/bytes/struct.Bytes.html#method.copy_from_slice) which does exactly what you are doing here internally.

Reply:
Addressed in a33367f0febdd930dfb5cba3892af203fc5b3a51